src/components/GuestWelcomeWindow.vue

```diff
@@ -153,6 +152,9 @@ export default {
 
 .description {
 	margin-bottom: 12px;
+	max-height: 2lh;
```

Comment:
Aren't two lines too little for a description? There could be some important information splitted in paragraphs
Maybe like this:
```css
max-height: 8ln
overflow-x: hidden
overflow-y: auto
```